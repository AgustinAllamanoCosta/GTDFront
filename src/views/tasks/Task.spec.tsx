import React from 'react';
import { expect } from '@jest/globals';
import { render } from '@testing-library/react';
import TaskView from './Task';
import { BrowserRouter } from 'react-router-dom';

describe('Task View', () => {
  test('Should render a login view with singin and login buttons', async () => {
    const { container } = render(
      <BrowserRouter>
        <TaskView />
      </BrowserRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
