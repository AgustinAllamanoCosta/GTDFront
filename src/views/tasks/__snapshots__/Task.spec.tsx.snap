// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Task View Should render a login view with singin and login buttons 1`] = `
<div>
  <div
    class="sc-QdHZx pJhcI"
  >
    <div
      class="sc-hbwYeq ejWnqu"
    >
      <div
        class="sc-hfDuwB ldHbDs"
        data-cy="Card-Contents"
        is_center="undefined"
        padding="false"
      >
        <div
          class="sc-hdxXtv DAwqi"
        >
          <span
            class="sc-evVYyG NaZUL"
            data-cy="Card-title"
          >
            Active Task
          </span>
          <span
            class="sc-evVYyG NaZUL"
            data-cy="Card-label"
          >
            0/3
          </span>
        </div>
        <div
          class="sc-kbUMEY geLPum"
          data-cy="Active-task-list"
        />
      </div>
    </div>
    <div
      class="sc-lfmRZu fwEZKN"
    >
      <div
        class="sc-hfDuwB ldHbDs"
        data-cy="Card-Contents"
        is_center="undefined"
        padding="false"
      >
        <div
          class="sc-hdxXtv DAwqi"
        >
          <span
            class="sc-evVYyG NaZUL"
            data-cy="Card-title"
          >
            Task Inbox
          </span>
          <span
            class="sc-evVYyG NaZUL"
            data-cy="Card-label"
          >
            total 0
          </span>
        </div>
        <div
          class="sc-dasNVf irclbw"
        />
        <div
          class="sc-cpKBeH sc-ZHFQk fiLIYm UcFRe"
          data-cy="task-add-button"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-plus sc-cYUugm gftMPA"
            data-icon="plus"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M256 80c0-17.7-14.3-32-32-32s-32 14.3-32 32V224H48c-17.7 0-32 14.3-32 32s14.3 32 32 32H192V432c0 17.7 14.3 32 32 32s32-14.3 32-32V288H400c17.7 0 32-14.3 32-32s-14.3-32-32-32H256V80z"
              fill="currentColor"
            />
          </svg>
          <input
            class="sc-hDqSpx jBVQaa"
            data-cy="task-add-button-input"
            placeholder="Add Task"
            value=""
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
